src/main/java/org/opentripplanner/middleware/models/AbstractUser.java

```diff
@@ -30,6 +30,13 @@
      * value, so the stored user will contain the value from Auth0 (e.g., "auth0|abcd1234").
      */
     public String auth0UserId = UUID.randomUUID().toString();
+
+    /**
+     * Random api key for testing.
+     */
+    public String apiKey = UUID.randomUUID().toString();
```

Comment:
I don't think this should this be a field on `AbstractUser`? It will get added to every user type, which doesn't seem like a good idea.

Reply:
Removed.